help/en/releasenotes/current-draft-note.shtml

```diff
@@ -465,7 +465,9 @@
     <h3>Preferences</h3>
         <a id="Preferences" name="Preferences"></a>
         <ul>
-            <li></li>
+            <li>Do not create blank profiles.
+            When deleting blank profiles do not remove directory.
+            Explicitly warn that a directory and its contents will be deleted when deleting a profile.</li>
```

Comment:
No need to change. But since this is HTML, everything will be on an single line. If you want to have three rows, you can add `<br>` at the end of the two first lines.